src/server/index.jsx

```diff
@@ -53,6 +54,12 @@ const server = express();
  * Default headers, compression, logging, status route
  */
 
+const getBuildMetadata = () => {
+  const data = JSON.parse(fs.readFileSync('package.json', 'utf8'));
```

Comment:
could just be `const { buildMetadata } = JSON.parse(fs.readFileSync('package.json', 'utf8'));`

Reply:
will make this change now

Reply:
Change made @jroebu14